python-package/lightgbm/dask.py

```diff
@@ -467,7 +520,7 @@ def predict(self, X, **kwargs):
 
     predict.__doc__ = LGBMClassifier.predict.__doc__
 
-    def predict_proba(self, X, **kwargs):
+    def predict_proba(self, X: _DaskCollection, **kwargs: Any) -> _DaskCollection:
```

Comment:
Based on https://github.com/microsoft/LightGBM/pull/3866/files#r565691112.
```suggestion
    def predict_proba(self, X: _DaskCollection, **kwargs: Any) -> dask_Array:
```